pages/common/clang-format.md

```diff
@@ -13,12 +13,12 @@
 
 - Format a file using a predefined coding style:
```

Comment:
```suggestion
- Format a file using a specific predefined coding style:
```

Reply:
This change must be done in another PR by somebody else. I approve this PR with keep this note in mind.